packages/corvid-cli/test/clone.e2e.js

```diff
@@ -401,4 +402,34 @@ describe("clone", () => {
       );
     });
   });
+  describe("Failed clone", () => {
+    it("should clean files", async () => {
+      const paths = ["src", ".corvid"];
+      editorServer = await localFakeEditorServer.start();
+      const { tempDir, siteUrl } = await setupSuccessfullClone();
+      process.env.QUERY = "failOnClone=true";
```

Comment:
this env variable will remain here for the next tests and will make them all fail
please find a better way for making the clone fail

Reply:
done